src/core/streams/stream_actions.ts

```diff
@@ -31,6 +31,9 @@ export const StreamActions: TurboStreamActions = {
   },
 
   update() {
-    this.targetElements.forEach((e) => e.replaceChildren(this.templateContent))
+    this.targetElements.forEach((targetElement) => {
+      targetElement.innerHTML = ""
+      targetElement.append(this.templateContent)
```

Comment:
@MaxHatfull if we're going to go forward with rolling back the `replaceChildren` calls, what do you think about implementing update in terms of the [polyfill](https://github.com/github/browser-support/pull/4/files#diff-08a775c335de1875f6d61c8046ccef7b5f90744fac68d6bc9621b8526b77514eR1-R4):

```typescript
export function replaceChildren(this: Element | Document | DocumentFragment, ...children: Node[]) {
   while (this.firstChild) this.removeChild(this.firstChild)
   this.append(...children)
 }
```

Reply:
I did think about that, the reason I didn't do that in this PR is to minimise the blast radius of this change. If you think this won't affect the projects of the people using hotwire and you think this is clearer, then I think it's a good idea. There's also the question of "is this simpler?" which I don't think it is as unsetting the innerHTML and setting the child to the template is what I'd expect to see when investigating a method called "update".